.github/npm-publish.yml

```diff
@@ -0,0 +1,39 @@
+name: NPM Registry Publication
+
+on:
+  push:
+    tags:
+      - node-[0-9]+.[0-9]+.[0-9]+
+
+jobs:
+  publish:
+    runs-on: ubuntu-latest
+
+    defaults:
+      run:
+        working-directory: node
+
+    strategy:
+      matrix:
+        node-version:
+         - 16.x
```

Comment:
I think we need to define what version of node we support.
I think do we support only 14 for `scalardl-node-client-sdk`?

Reply:
@feeblefakie 

> I think we need to define what version of node we support.
> I think do we support only 14 for scalardl-node-client-sdk?

Understood. Let's discuss about this.

scalardl-node-client-sdk now supports only Node.js version 16 (the active LTS).
(Thanks for your reminder. I noticed I have a typo [here](https://github.com/scalar-labs/scalardl-node-client-sdk/blob/master/package.json#L8). I will create another PR to fix it)

Reply:
> I think we need to define what version of node we support.

Discussed with Hiro.
We will support all active and maintenance LTS versions as long as the cost isn't too high (for example, no need to have additional codes for supporting various versions)